### DISCUSSION TITLE
A bug

### DISCUSSION THREAD

**Discussion by Victory-Hugo** [Kanaries/pygwalker#486] [Q&A]
I don't know why, but if I change the default theme color, for example from blue to green, it crashes. Why is this?
<img width="145" alt="PixPin_2024-03-18_09-49-20" src="https://github.com/Kanaries/pygwalker/assets/143314364/ba7053f7-3a2c-415c-8f32-aad48d14a884">
<img width="229" alt="PixPin_2024-03-18_09-49-29" src="https://github.com/Kanaries/pygwalker/assets/143314364/9e3dd597-7c02-4aa2-adcd-a735d836a3c7">

**Comment by ObservedObserver** [comment]
Hi @Victory-Hugo, I tried to reproduce the bug but it still works fine. Here is my step-by-step operations.

https://github.com/Kanaries/pygwalker/assets/22167673/a0c947ea-7397-4205-9438-ac5b24c2078c

I am using the latest version of pygwalker (0.4.7).

I would suggest you can try to restart the jupyter kernel and re-run pygwalker to try again. If the issue still exists, please share more details include:
- local version of pygwalker
- browser you are using
- local python version
